example_inputs/fp_multiplier/Makefile

```diff
@@ -0,0 +1,17 @@
+#    Copyright 2022 Google LLC
```

Comment:
You seem to have a whole bunch of extra spaces at the front of the copyright?

Reply:
done